_bleio/scan_entry.py

```diff
@@ -69,8 +69,9 @@ def _from_bleak(cls, device: BLEDevice) -> "ScanEntry":
         )
 
     def matches(
-        self, prefixes: bytes, all: bool = True  # pylint: disable=redefined-builtin
-    ) -> bool:
+        self,
+        prefixes: bytes,
+        match_all: bool = True,
```

Comment:
```suggestion
        match_all: bool = True,
    ) -> bool:
```
unintentional deletion from me, maybe